sources/packages/web/src/assets/css/base.scss

```diff
@@ -38,6 +38,7 @@ html * {
 }
 
 .content-group {
+  color: $app-content-color;
   border: 1px solid #e8e8e8;
```

Comment:
Not part of the PR. Since you are working on this CSS. Pls replace #e8e8e8; with global variable